user-interface/src/lib/components/CaseNumberInput.tsx

```diff
@@ -24,7 +24,7 @@ type CaseNumberInputProps = Omit<InputProps, 'onChange'> & {
 };
 
 function CaseNumberInputComponent(props: CaseNumberInputProps, ref: React.Ref<InputRef>) {
-  const { onChange, allowEnterKey, allowPartialCaseNumber } = props;
+  const { onChange, allowEnterKey, allowPartialCaseNumber, ...otherProps } = props;
```

Comment:
I remember commenting something similar in another PR about using a spread of `...inputProps` versus `...otherProps`